make_sleuth_folders.py

```diff
@@ -1,61 +1,76 @@
+"""This utility creates directories for the Sleuth files for University
+of London's 'Intro to Programming I' course.
+
+Run the program in the folder where you want to create the directories.
+You can create all the directories, or just the rookie/pro stages."""
+
 import os
 
 # get current directory
-currentPath = os.getcwd()
+CURRENT_PATH = os.getcwd()
+SUB_PATHS = [1, 2, 3, 4]
+
+
+def create_structure():

```

Comment:
Since this code is now inside a function, the script simply reads the functions and won't execute anything unless it is told to.